include/RAJA/internal/exec-cuda/forall_cuda.hxx

```diff
@@ -61,13 +61,13 @@
 //
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
 
-#include "RAJA/int_datatypes.hxx"
+#include "RAJA/Types.hxx"
```

Comment:
We should settle on a common case/naming scheme for header files. Let's do this when the dust settles.  I'll add an issue.